src/index.css

```diff
@@ -10,13 +10,42 @@ form .field-aligner {
 	max-width: 80%;
 }
 
-#root, #App {
+#App {
   height: 100%;
   display: flex;
```

Comment:
I don't think this ever got addressed, but we may want to find out which versions of what browsers our beta tests are using. `flex` is relatively new and won't work in some.

Reply:
IIRC, Andrew said that he would send a notice to the folks we are demoing to, requiring fresh Chrome installs before we demo / test.  So, addressed on a process level :)

Reply:
lol, that's one way to do it!

Reply:
Should work on about [85% of the browsers people use](https://caniuse.com/#feat=flexbox). Not ideal, not the end of the world either. I like flexbox because it makes otherwise hacky solutions not hacky, but it being a relatively new feature is a downside.

Reply:
Yeah. My concern is because I've specifically heard that organizations with little funding tend to have outdated systems and tend to rely on older technology, but maybe that's not the case here.